packages/sdk-browser/README.md

```diff
@@ -12,27 +12,126 @@ Turnkey API documentation lives here: https://docs.turnkey.com.
 $ npm install @turnkey/sdk-browser
 ```
 
+### Initialize
+
+```typescript
+import { Turnkey } from "@turnkey/sdk-browser";
+
+const turnkey = new Turnkey({
+  apiBaseUrl: "https://api.turnkey.com",
+  defaultOrganizationId: process.env.TURNKEY_ORGANIZATION_ID,
+  // Optional: Your relying party ID - for use with Passkey authentication
+  rpId: process.env.TURNKEY_RP_ID,
+});
+```
+
+### Turnkey Clients
+
+#### Passkey
+
+The Passkey client allows for authentication to Turnkey's API using Passkeys.
+
+```typescript
+const passkeyClient = turnkey.passkeyClient();
+
+// User will be prompted to login with their passkey
+await passkeyClient.login();
```

Comment:
do we want to describe what exactly login is doing? or perhaps this can wait for Omkar's work on documenting all these methods

cc @omkarshanbhag as some of these methods will be getting tweaked